ramos/compat.py

```diff
@@ -1,8 +1,10 @@
 try:
     from django.conf import settings  # noqa
 except ImportError:
-    from simple_settings import settings  # noqa
-
+    try:
+        from simple_settings import settings  # noqa
+    except ImportError:
+        settings = None  # noqa
```

Comment:
What about adding test scenarios for this case? There is no coverage pointed out, it would be nice to have a test describing its behaviour.